exonum/src/blockchain/service.rs

```diff
@@ -167,9 +172,9 @@ pub trait Service: Send + Sync + 'static {
     /// these operations should be performed in the `Transaction::verify` and
     /// `Transaction::execute` methods.
     ///
-    /// `transactions!` macro generates code that allows simple implementation, see
+    /// `#[derive(TransactionSet)]` macro generates code that allows simple implementation, see
```

Comment:
@dvush `#[derive...` is an attribute. I guess this section of doc should be changed.